practical/python/server/batch_roi_export_to_table.py

```diff
@@ -246,13 +246,15 @@ def save_table(conn, images, image_data, script_params, project=None):
     data = [img_column] + cols
     table.initialize(data)
     table.addData(data)
-    table.close()
 
     if project is None:
         log("No Project found to link table")
     else:
         link_table(conn, table, project)
 
+    # close after table.getOriginalFile()
```

Comment:
This script has been migrated to https://github.com/ome/omero-guide-python/blob/master/scripts/batch_roi_export_to_table.py
The changes should be made in the omero-guide-python repository

Reply:
Fix ported to https://github.com/ome/omero-guide-python/pull/33 and the file deleted from this repo (links to it were already pointing to `omero-guide-python` repo.